src/docs/goose-book/src/config/defaults.md

```diff
@@ -67,7 +67,7 @@ For example, without any run-time options the following load test would automati
         )
         .set_default(GooseDefault::Host, "local.dev")?
         .set_default(GooseDefault::RequestsFile, "goose-requests.log")?
```

Comment:
Can you also fix this? It should be [`GooseDefault::RequestLog`](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestLog)

Reply:
I might have to re-fork, create a branch and add in my changes and all your suggestions. Would this be a problem? Then link the new  PR to this one? Until I figure out the correct way of doing this? Be tomorrow, now though.